docs/General/Manuals/client-manual.rst

```diff
@@ -654,14 +654,199 @@ Advanced Unit Selection
 Unit Menu
 ^^^^^^^^^
 
+The :guilabel:`Unit` menu is used to give units orders.
```

Comment:
Side note: We need to fix this to not show unavailable options, and not just grey them out. Greying is cluttering the UI too much.

Reply:
Then, combat and work can mostly be merged together. Since very few units have so many orders available.

Reply:
Issue #1235